libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/ForegroundService.kt

```diff
@@ -62,14 +64,27 @@ abstract class ForegroundService : SdkForegroundService, EventListener, Service(
     }
 
     /** Stop the service */
+    private val serviceTimeoutHandler = Handler(Looper.getMainLooper())
+    private val serviceTimeoutRunnable: Runnable = Runnable {
```

Comment:
@dangeross can you please elaborate when do we extend the serviceTimeout vs where we extend the shutdownHandler ?

Reply:
The serviceTimeout is extended (reset to 3 mins) on each call to `onStartCommand`. The shutdownHandler is stopped on a call to `onStartCommand` and started when the last job is complete. So the shutdownHandler keeps the service "warm" for 1 min before shutting down.

Reply:
Thanks for the explanation.
I see here that when a running service calls startForegroundService it gets an extended running period of another 3 minutes: https://developer.android.com/about/versions/14/changes/fgs-types-required#short-service
Do we use this mechanism?

Reply:
Yes, it's called in `onStartCommand`

Reply:
I think I have to change the implementation to use
```
ServiceCompat.startForeground(0, notification, FOREGROUND_SERVICE_TYPE_SHORT_SERVICE)
```

Reply:
After testing this, just `startForeground(0, notification)` is sufficient. Also ServiceCompat.startForeground is not available in API level 33